postgresql/restore.sh

```diff
@@ -6,13 +6,13 @@
 
 # Before running the restore, you'll have to stop the ECS task to close any open connections.
 # Then, run the following to drop the database.
-# psql -U concordia -h $POSTGRES_HOST
+# psql -U concordia -h $POSTGRESQL_HOST
 # \c postgres
 # drop database concordia;
 
-export POSTGRES_HOST=
+export POSTGRESQL_HOST=
```

Comment:
Wouldn't it just be better to either delete this, requiring you to set the variable first, or have it use `$POSTGRESQL_HOST` with a default `localhost`?

Reply:
I guess a localhost default would make sense. The restore process is interactive so this file is really meant more to remind whoever is doing it of what needs to be there.

Reply:
The main time I notice this is when these two scripts prevent docker-compose from working until I delete them

Reply:
Ah, I need to delete and re-create my db container.

Reply:
I'm not able to reproduce the issue you described. I don't see any place where anything is referring to the dump or restore scripts. I thought I was the only one using them for loading prod data to dev, test and stage.

Reply:
The default postgres container automatically runs `postgresql/*.sh`.

Reply:
```
Creating concordia_db_1 ... done
Attaching to concordia_db_1
db_1          | The files belonging to this database system will be owned by user "postgres".
db_1          | This user must also own the server process.
db_1          | 
db_1          | The database cluster will be initialized with locale "en_US.utf8".
db_1          | The default database encoding has accordingly been set to "UTF8".
db_1          | The default text search configuration will be set to "english".
db_1          | 
db_1          | Data page checksums are disabled.
db_1          | 
db_1          | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db_1          | creating subdirectories ... ok
db_1          | selecting default max_connections ... 100
db_1          | selecting default shared_buffers ... 128MB
db_1          | selecting dynamic shared memory implementation ... posix
db_1          | creating configuration files ... ok
db_1          | running bootstrap script ... ok
db_1          | performing post-bootstrap initialization ... ok
db_1          | syncing data to disk ... ok
db_1          | 
db_1          | Success. You can now start the database server using:
db_1          | 
db_1          |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db_1          | 
db_1          | 
db_1          | WARNING: enabling "trust" authentication for local connections
db_1          | You can change this by editing pg_hba.conf or using the option -A, or
db_1          | --auth-local and --auth-host, the next time you run initdb.
db_1          | waiting for server to start....2019-01-03 14:59:22.980 UTC [44] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db_1          | 2019-01-03 14:59:22.989 UTC [45] LOG:  database system was shut down at 2019-01-03 14:59:22 UTC
db_1          | 2019-01-03 14:59:22.993 UTC [44] LOG:  database system is ready to accept connections
db_1          |  done
db_1          | server started
db_1          | CREATE DATABASE
db_1          | 
db_1          | 
db_1          | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/create-multiple-postgresql-databases.sh
db_1          | Multiple database creation requested: test_concordia
db_1          |   Creating user and database 'test_concordia'
db_1          | CREATE ROLE
db_1          | CREATE DATABASE
db_1          | GRANT
db_1          | Multiple databases created
db_1          | 
db_1          | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/dump.sh
db_1          | /docker-entrypoint-initdb.d/dump.sh: line 6: POSTGRES_HOST: unbound variable
concordia_db_1 exited with code 1
```

Reply:
Gotcha. I moved 'em. It doesn't even make sense to have both dump and restore scripts being executed when the container is being created anyway.